### PR TITLE
[F] 10 create code of conduct

### PR DESCRIPTION
**Pull request by John4850**
fixes Create Code of Conduct

### CODE REVIEW

**Comment by John4850** [comment]
#10 
Screen Shot 2023-05-26 at 4 08 03 PM
Screen Shot 2023-05-26 at 4 07 57 PM

File: src/components/index.ts [hunk 1]
```diff
@@ -4,3 +4,4 @@ export { Header } from "./header/Header"
 export { LinkButton } from "./linkButton/LinkButton"
 export { SplashText } from "./splashText/SplashText"
 export { TextBlock } from "./textBlock/TextBlock"
+export { CodeOfConductText } from "./codeOfConductText/CodeOfConductText"
```
Comment (MollyJeanB) [inline src/components/index.ts:7]: Let's keep these imports in alphabetical order

File: src/pages/code-of-conduct.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { TextBlock, CodeOfConductText } from "@/components"
+import Link from 'next/link';
+
+export default function CodeOfConduct() {
+  return (
+    <>
+
+      <Head>
+        <title>Empathetech Code of Conduct</title>
+        <meta name="description" content="code of conduct for empathetech" />
```
Comment (MollyJeanB) [inline src/pages/code-of-conduct.tsx:13]: Let's capitalize "Code" here in the content description

File: src/styles/page.module.css [hunk 3]
```diff
@@ -15,3 +15,10 @@
 	align-items: center;
 	padding-left: var(--size-xs);
 }
+
+.nav {
```
Comment (MollyJeanB) [inline src/styles/page.module.css:19]: Style nitpick that you can feel free to push back on: I prefer not to use abbreviations in variable or classnames (this one is innocuous but some can really make the codebase less readable over time). Can you do `.navigation` instead?

**Review by MollyJeanB — CHANGES_REQUESTED** [review]
Looks great! Just a few minor notes

File: src/components/index.ts (hunk 1 above)
Comment (John4850) [reply src/components/index.ts]: 👍

File: src/pages/code-of-conduct.tsx (hunk 2 above)
Comment (John4850) [reply src/pages/code-of-conduct.tsx:13]:
```
<meta name="description" content="Code of conduct for empathetech" />
```
Like so? Should Empathetech be caps too?

**Review by John4850 — COMMENTED** [review]
changes made

**Review by MollyJeanB — COMMENTED** [review]
Looks great! Before you merge, will you squash these commits into a single commit?

**Review by MollyJeanB — APPROVED** [review]

**Pull request merged**